<launch>
    <!-- <node pkg="drone" type="navigation.py" name="talker_node"></node> -->
    <node pkg="drone" type="inner_control.py" name="outer_control_node"></node>
    <node pkg="rosserial_python" type="serial_node.py" name="arduino_node">
        <param name="port" value="/dev/ttyUSB0"/>
    </node>
</launch>